front/src/Components/ChangePassword/ChangePassword.js

```diff
@@ -1,131 +0,0 @@
-import { Body, Page, InnerFieldset, AddButton } from "../../Styles/GlobalStyles.js";
```

Comment:
Uma sugestão de estrutura a se seguir no frontend seria essa:

https://dev.to/mmcshinsky/why-frontend-architecture-matters-1ldj

Nela, separamos os componentes destinados a estrutura de página dos componentes menores e compartilhados em mais de um lugar. A ideia, assim como uma estrutura de backend, é garantir a semântica e organização do projeto.